internal/cmd/add.go

```diff
@@ -0,0 +1,41 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/keygen-sh/keygen-relay/internal/licenses"
+	"github.com/spf13/cobra"
+)
+
+func AddCmd(manager licenses.Manager) *cobra.Command {
+	var (
+		filePath  string
+		key       string
+		publicKey string
+	)
+
+	cmd := &cobra.Command{
```

Comment:
Can we add `SilenceUsage: true` to each command? The output on error is very verbose and it's hard to determine what the actual issue is because there's so much going on.

Reply:
Thank you for that! I have added it

It really looks better :)